src/dinterpret.d

```diff
@@ -2022,7 +2022,7 @@ public:
         }
         if (goal == ctfeNeedLvalue)
         {
-            if (istate.fd.vthis)
```

Comment:
It is suspicious that istate can be null in here.
and indicates a lager issue.
Please add a TODO or FIXME stating it.